docs/releases/1.9.txt

```diff
@@ -483,6 +483,7 @@ CSRF
 
 * The request header's name used for CSRF authentication can be customized
   with :setting:`CSRF_HEADER_NAME`.
+* The setting :setting:`CSRF_TRUSTED_ORIGINS` was added to allow certain cross origin requests under HTTPS.
```

Comment:
I think we can cut out "certain" here and add "POST" -- "added to allow cross origin POST requests under HTTPS."